src/HTMLImports/path.js

```diff
@@ -11,27 +11,31 @@ HTMLImports.addModule(function(scope) {
 
 var CSS_URL_REGEXP = /(url\()([^)]*)(\))/g;
 var CSS_IMPORT_REGEXP = /(@import[\s]+(?!url\())([^;]*)(;)/g;
+var CSS_ABSOLUTE_URL_REGEXP = /^(http[s]?:)?\/\//i
 
 // path fixup: style elements in imports must be made relative to the main
 // document. We fixup url's in url() and @import.
 var path = {
 
-  resolveUrlsInStyle: function(style) {
+  resolveUrlsInStyle: function(style, linkUrl) {
     var doc = style.ownerDocument;
     var resolver = doc.createElement('a');
-    style.textContent = this.resolveUrlsInCssText(style.textContent, resolver);
+    style.textContent = this.resolveUrlsInCssText(style.textContent, linkUrl, resolver);
     return style;
   },
 
-  resolveUrlsInCssText: function(cssText, urlObj) {
-    var r = this.replaceUrls(cssText, urlObj, CSS_URL_REGEXP);
-    r = this.replaceUrls(r, urlObj, CSS_IMPORT_REGEXP);
+  resolveUrlsInCssText: function(cssText, linkUrl, urlObj) {
+    var r = this.replaceUrls(cssText, urlObj, linkUrl, CSS_URL_REGEXP);
+    r = this.replaceUrls(r, urlObj, linkUrl, CSS_IMPORT_REGEXP);
     return r;
   },
 
-  replaceUrls: function(text, urlObj, regexp) {
+  replaceUrls: function(text, urlObj, linkUrl, regexp) {
     return text.replace(regexp, function(m, pre, url, post) {
       var urlPath = url.replace(/["']/g, '');
+      if (!CSS_ABSOLUTE_URL_REGEXP.test(urlPath)) {
```

Comment:
if the URL is parsed using the URL object, it should be easy to check for an absolute path, since at that point the URL is parsed (the parse function at https://github.com/Polymer/URL/blob/master/url.js#L83 ... also it handles more schemes)